src/payment/payment-methods/cybersource/service.js

```diff
@@ -79,7 +79,30 @@ function getPaymentToken(microformOptions) {
 export async function checkoutWithToken(basket, { cardHolderInfo, cardDetails }) {
   const { basketId } = basket;
 
-  const paymentToken = await getPaymentToken(cardDetails);
+  const paymentToken = await getPaymentToken(cardDetails).catch((error) => {
+    if (error.reason !== 'CREATE_TOKEN_VALIDATION_FIELDS') {
```

Comment:
[optional] wondering aloud if there are other types of errors that should produce user-visible feedback?

Reply:
https://developer.cybersource.com/api/developer-guides/dita-flex/SAFlexibleToken/FlexMicroform/api_reference/class_MicroformError.html has the errors. I didn't see any others that looked user-facing.